pyroma/distributiondata.py

```diff
@@ -12,6 +13,16 @@
 from pyroma import projectdata
 
 
+def _safe_extract_tar(tar, path=".", members=None, numeric_owner=False):
+    """Safely extract a tar w/o traversing parent dirs to fix CVE-2007-4559."""
+    for member in tar.getmembers():
+        root = pathlib.Path(path).resolve()
```

Comment:
This line could be outside, the loop, I think?
Not a big deal, I know, but still.

Reply:
Yup, great point—missed that when I was refactoring this to use `pathlib`. I've pushed an update with this fixed, thanks.